### PR TITLE
Feature/tc 90

### PR DESCRIPTION
**Pull request by pangeon**
I fixed loombok problems in my IDE. Now nothing shows errors.

### CODE REVIEW

File: src/main/java/codeenthusiast/TrainingCenterApp/muscle/MuscleDTO.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
     private String name;
     private String description;
+    private String imageUrl;
     private List<Image> images;
```
Comment (naslakboss) [inline src/main/java/codeenthusiast/TrainingCenterApp/muscle/MuscleDTO.java:19]: Muscle entity could have multiple images. for example -> front and back view.
Change string imageUrl to List<String> imageUrls and delete List<Image>
Appropiate mapping in MuscleMapper will be necessary also

File: src/main/java/codeenthusiast/TrainingCenterApp/user/UserServiceImpl.java [hunk 2]
```diff
@@ -15,7 +16,7 @@
     private UserMapper userMapper;
     private UserRepository userRepository;
 
-    public UserServiceImpl(AbstractRepository<User> repository, AbstractMapper<User,
+    public UserServiceImpl(@Qualifier("userRepository") AbstractRepository<User> repository, AbstractMapper<User,
```
Comment (naslakboss) [inline src/main/java/codeenthusiast/TrainingCenterApp/user/UserServiceImpl.java:19]: Why you used @Qualifier here?

**Review by naslakboss — COMMENTED** [review]
Very good commit, you just need to : 
1) change some data in DTO
2) add this new methods also in controller

File: src/main/java/codeenthusiast/TrainingCenterApp/muscle/MuscleDTO.java (hunk 1 above)
Comment (pangeon) [reply src/main/java/codeenthusiast/TrainingCenterApp/muscle/MuscleDTO.java:19]: I don't understand when I delete **List<Image> images;** Muscle.class when breaks.

Comment (naslakboss) [reply src/main/java/codeenthusiast/TrainingCenterApp/muscle/MuscleDTO.java:19]: breaks ?

**Pull request closed without merging**